README.md

```diff
@@ -52,6 +55,11 @@ You need the following permissions to run this module.
 - Account Management
     - Security and Compliance Center service
         - `Administrator` access
+- IAM Services
+    - Cloud Object Storage
+        - `Reader` access
+    - Event Notifications
+        - `Reader` access
```

Comment:
These aren't needed to run the module are they? SCC will use the service to service auth policy for this communication. I guess whatever access is needed to create the auth policy should be listed here